client/src/components/Header/components/universal-nav.tsx

```diff
@@ -75,14 +77,25 @@ export const UniversalNav = ({
           </div>
         ) : (
           <>
+            <Media minWidth={DONATE_NAV_EXPOSED_WIDTH + 1}>
+              {' '}
+              <Link
+                sameTab={false}
+                to='/donate'
+                data-test-label='nav-donate-button'
```

Comment:
Recently at my company we started migrating away from test specific labels such as this in favor of aria labels. Aria labels should be able to serve the same purpose in tests but they also are important for accessibility. What are your thoughts?

Reply:
The accessible name for the link will be the text between the `<Link>` and `</Link>` tags, so adding an `aria-label` to this link is not only not needed for accessibility but also considered an anti-pattern.